Makefile

```diff
@@ -20,6 +20,10 @@ unit: ## Run all unit tests with coverage report
 integration-tests:  ## Run integration tests
 	go test -tags integration -v ./integration/
 
+.PHONY: e2e-tests
+e2e-tests: ## Run e2e tests
```

Comment:
should this have a dependency on `build-docker-image` to create an image if it doesn't exist?

Reply:
how about we check if the image is built in the `/run_docker.sh` script instead? Otherwise that script will not be super useful to run the tests locally.